packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -296,18 +304,27 @@ export const prepareVaultManagerKit = (
             'MakeVault',
           );
         },
+        /**
+         * @deprecated use getPublicTopics
+         */
         getSubscriber() {
-          return storedAssetSubscriber;
+          return assetSubscriber;
         },
+        /**
+         * @deprecated use getPublicTopics
+         */
         getMetrics() {
-          return storedMetricsSubscriber;
+          return metricsSubscriber;
         },
         getQuotes() {
           return storedQuotesNotifier;
         },
         getCompoundedInterest() {
           return this.state.compoundedInterest;
         },
+        getPublicTopics() {
```

Comment:
There should be a test for this.

Reply:
it's covered by the calls to [topicPath](https://github.com/Agoric/agoric-sdk/blob/9883ed6937ac5b0fd73fabb5073c85ce93afd346/packages/inter-protocol/test/supports.js#L211-L220) in the 'storage keys' test.